apps/explorer/src/pages/transaction-result/TransactionResult.tsx

```diff
@@ -1,19 +1,77 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { ThumbUpFill32 } from '@mysten/icons';
+import { type SuiTransactionBlockResponse } from '@mysten/sui.js';
 import { LoadingIndicator } from '@mysten/ui';
+import clsx from 'clsx';
 import { useParams } from 'react-router-dom';
 
 import { TransactionView } from './TransactionView';
 import { PageLayout } from '~/components/Layout/PageLayout';
 import { useGetTransaction } from '~/hooks/useGetTransaction';
 import { Banner } from '~/ui/Banner';
+import { PageHeader } from '~/ui/PageHeader';
+
+function StatusIcon({ success }: { success: boolean }) {
```

Comment:
This component could be move to it's own file. It could be useful in other places maybe